.NET/Microsoft.Recognizers.Text.DateTime/Italian/Extractors/ItalianDatePeriodExtractorConfiguration.cs

```diff
@@ -0,0 +1,325 @@
+                DateTimeDefinitions.EngMonthRegex,
+                RegexOptions.IgnoreCase | RegexOptions.Singleline);
+
+        public static readonly Regex MonthSuffixRegex =
+            new Regex(
+                DateTimeDefinitions.MonthSuffixRegex, // in, of, no "on"...
+                RegexOptions.IgnoreCase | RegexOptions.Singleline);
+
+        public static readonly Regex DateUnitRegex = new Regex(
+            DateTimeDefinitions.DateUnitRegex, // year, month, week, day
+            RegexOptions.IgnoreCase | RegexOptions.Singleline);
+
+        public static readonly Regex PastPrefixRegex = // **In Italian, Past/Next is suffix, but interface enforces this 
+            new Regex(
+                DateTimeDefinitions.PastSuffixRegex, // past, last, previous
+                RegexOptions.IgnoreCase | RegexOptions.Singleline);
+
+        public static readonly Regex NextPrefixRegex = // **In Italian, Past/Next is suffix, but interface enforces this 
+            new Regex(
+                DateTimeDefinitions.NextSuffixRegex, // next, in
+                RegexOptions.IgnoreCase | RegexOptions.Singleline);
+
+        public static readonly Regex FutureSuffixRegex =
+            new Regex(DateTimeDefinitions.FutureSuffixRegex,
+                RegexOptions.IgnoreCase | RegexOptions.Singleline);
+
+        public static readonly Regex ThisPrefexRegex =
+            new Regex(
+                DateTimeDefinitions.ThisPrefixRegex,
+                RegexOptions.IgnoreCase | RegexOptions.Singleline);
+
+        // composite regexes
+        public static readonly Regex SimpleCasesRegex =
+            new Regex(
+                DateTimeDefinitions.SimpleCasesRegex,
+                RegexOptions.IgnoreCase | RegexOptions.Singleline);
+
+        public static readonly Regex MonthFrontSimpleCasesRegex = 
+            new Regex(
+                DateTimeDefinitions.MonthFrontSimpleCasesRegex, // between 'x' until 'y', from 'x' until 'y'
+                RegexOptions.IgnoreCase | RegexOptions.Singleline);
+
+        public static readonly Regex MonthFrontBetweenRegex =
+            new Regex(
+                DateTimeDefinitions.MonthFrontBetweenRegex,
+                RegexOptions.IgnoreCase | RegexOptions.Singleline);
+
+        public static readonly Regex BetweenRegex =
+            new Regex(
+                DateTimeDefinitions.BetweenRegex,
+                RegexOptions.IgnoreCase | RegexOptions.Singleline);
+
+        public static readonly Regex MonthWithYear =
+            new Regex(
+                DateTimeDefinitions.MonthWithYear,
+                RegexOptions.IgnoreCase | RegexOptions.Singleline);
+
+        public static readonly Regex OneWordPeriodRegex =
+            new Regex(
+               DateTimeDefinitions.OneWordPeriodRegex,
+                RegexOptions.IgnoreCase | RegexOptions.Singleline); // a cote de - 'next to', cette - 'this', dernier - 'last' (always after the noun, i.e annee dernier - 'last year'  
```

Comment:
Nit: I'd prefer to move the code comment before the regex declaration.
It's easier to read that way.